setup.py

```diff
@@ -85,7 +85,8 @@
         'scrapbook', 
         'werkzeug==2.0.3',
         'sentry_sdk',
-        'rdflib'
+        'rdflib',
```

Comment:
might be better to restring `rdflib` version following https://github.com/oda-hub/dispatcher-app/pull/646

Reply:
This was done on master. I've merged the changes to this branch.